plaso/parsers/jsonl_plugins/gcp_log.py

```diff
@@ -99,6 +136,213 @@ def _ParseJSONPayload(self, json_dict, event_data):
     if actor_json:
       event_data.user = self._GetJSONValue(actor_json, 'user')
 
+  def _ParseAuthenticationInfo(self, proto_payload, event_data):
+    """Extracts information from `protoPayload.authenticationInfo`.
+
+    Args:
+      proto_payload (dict): JSON dictionary of the `protoPayload` value.
+      event_data (GCPLogEventData): event data.
+    """
+    if not proto_payload:
```

Comment:
Why not move this check to the location where _ParseAuthenticationInfo is called?